flutter/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://docs.sentry.io/platforms/flutter/
 repository: https://github.com/getsentry/sentry-dart
 
 environment:
-  sdk: ">=2.8.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
   flutter: ">=1.17.0"
```

Comment:
@marandaneto Does it make sense to keep it this low? I wonder if it causes any problems because Flutter 1.17 doesn't has null safety. On the other hand
[package_info](https://github.com/flutter/plugins/blob/b709f7e4994aa48d2be815d07a1da0d66180c86e/packages/package_info/pubspec.yaml) for example allows versions as low as 1.12 while also having null safety.

Reply:
yeah unless we have to, lets keep as it is, all the official plugins are down to 1.12

Reply:
btw looking at https://dart.dev/null-safety

> Sound null safety is available in Dart 2.12 and Flutter 2.

should we bump to min. v2?